### DISCUSSION TITLE
field expansions

### DISCUSSION THREAD

**Discussion by lstebner** [jeremyckahn/farmhand#86] [Ideas]
maybe instead of the field just getting larger, the player could actually add different plots. this would allow for a more infinite expansion of how many crops can be grown (doesn't have to be infinite but it will be easier to build a UI for many plots rather than a giant field). it also lets more organized players grow more organized crops, and makes the "range" setting a little less ridiculous (not that it matters really if it's believable or not, but as a player, i feel like i've done more work for the day if i water a few plots rather than water one giant one in one click). 

furthermore, this would change up how some items mare used like scarecrows, and fertilizer, which are applied to the whole field. now the player would be applying these per plot, which gives the player some more reason to buy and manage those resources.

last thought on this, the ideal solution may be something of a combination of adding plots, but also being able to enlarge them (maybe just once though).

**Comment by jeremyckahn** [comment]
I definitely think the game is ready for additional field expansion! I'm not sure if we want to support infinite expansion, as I could see it creating a lot of technical resource problems (unless we want to get really clever with the implementation). Assuming we don't, I think the simplest way to do this is to allow the user to buy additional fields in the Shop after they've purchased the largest one. Each new field could work as a sort of "sub page" of the Field page (think arrows on the side of the field screen that let you cycle through your fields).

More than anything, I just want the field expansion mechanic to be as simple as can be. What do you think @lstebner?

------------------------------

>last thought on this, the ideal solution may be something of a combination of adding plots, but also being able to enlarge them (maybe just once though).

Could you expand (🙃) on what you mean by enlarging a plot? How are you envisioning the gameplay for that idea?

**Reply by lstebner** [reply]
i'm sorry i missed your response until #105 made me check back in here :(. 

i think i wrote this up while i was still getting used to the terminology of things. what i'm calling "plots" is more so what is actually just the "field". so i think i was talking about having a combo of being able to expand fields, but also buy more fields (i'm imagining each one as an "acre") and then things like scarecrows can only cover 1 field/acre max, so you have to buy those per-field now. but each field can be enlarged like the mechanic you currently have, before being maxed out. i still do like the idea of more fields, but higher yield per plot may be an easier solution to allow more crop growth without dealing with the UI of multiple fields. and that's the connection back to #105 !

**Reply by jeremyckahn** [reply]
I think we're more or less on the same page in terms of design ideas! I'm on the fence about whether it's overcomplicating things to have variable sizes for "acres" beyond the first one. I was thinking that the player would upgrade the first acre as they currently do, and then buy additional acres only at the maxed-out size up to, say, 10 times. If we allow them to upgrade each acre independently, I think the UI to manage it could be a little confusing for the player. I don't feel super strongly about this and could be convinced to have independently-sized acres, though.

All that said, I also think it'll be easier to implement and design an increased yield per-plot first (#105). It sounds like we're converging on putting field expansion on the back burner for now (though certainly not killing the idea) in favor of field tool upgrades. Let's come back to this idea once mining and tool expansion are shipped and see how we feel about field expansion after that!
